C/cryptography/railFenceCipher.c

```diff
@@ -0,0 +1,64 @@
+/* The rail fence cipher (also called a zigzag cipher) is a form of transposition cipher. It derives its name from the way in which it is encoded. Given a plain-text message and a numeric key, it cipher the given text using Rail Fence algorithm(here key == 2). 
```

Comment:
Break this line down into multiple separate lines. That makes it easy for us to read and understand.